src/main/scala/cromwell/binding/WdlImplicits.scala

```diff
@@ -0,0 +1,9 @@
+package cromwell.binding
+
+import cromwell.binding.types.{WdlIntegerType, WdlStringType}
+
+object WdlImplicits {
+
+  implicit def stringToWdlValue(string: String): WdlValue = WdlValue(string, WdlStringType)
```

Comment:
Unless there's a reason to do otherwise I'd rather see these be implicit value classes with a toWdlValue method